server/src/main/java/com/hedera/block/server/Server.java

```diff
@@ -1,38 +1,90 @@
+ * limitations under the License.
+ */
+
 package com.hedera.block.server;
 
-import com.hedera.block.protos.EchoServiceGrpcProto;
+import com.hedera.block.protos.BlockStreamServiceGrpcProto;
+import com.hedera.block.server.mediator.LiveStreamMediatorImpl;
+import com.hedera.block.server.persistence.WriteThroughCacheHandler;
+import com.hedera.block.server.persistence.storage.BlockStorage;
+import com.hedera.block.server.persistence.storage.FileSystemBlockStorage;
+import io.grpc.stub.ServerCalls;
 import io.grpc.stub.StreamObserver;
+import io.helidon.config.Config;
 import io.helidon.webserver.WebServer;
 import io.helidon.webserver.grpc.GrpcRouting;
 import io.helidon.webserver.http.HttpRouting;
 
+import java.io.IOException;
+import java.util.stream.Stream;
+
+import static com.hedera.block.server.Constants.*;
+
 /**
  * Main class for the block node server
  */
 public class Server {
-    private Server() {
-        // Not meant to be instantiated
-    }
+
+    // Function stubs to satisfy the bidi routing param signatures.  The implementations are in the service class.
+    private static ServerCalls.BidiStreamingMethod<Stream<BlockStreamServiceGrpcProto.Block>, StreamObserver<BlockStreamServiceGrpcProto.Block>> clientBidiStreamingMethod;
+    private static ServerCalls.BidiStreamingMethod<Stream<BlockStreamServiceGrpcProto.BlockResponse>, StreamObserver<BlockStreamServiceGrpcProto.Block>> serverBidiStreamingMethod;
+
+    private static final System.Logger LOGGER = System.getLogger(Server.class.getName());
+
+    private Server() {}
 
     /**
      * Main entrypoint for the block node server
      *
      * @param args Command line arguments. Not used at present,
      */
-    public static void main(String[] args) {
-        WebServer.builder()
-                .port(8080)
-                .addRouting(HttpRouting.builder()
-                        .get("/greet", (req, res) -> res.send("Hello World!")))
-                .addRouting(GrpcRouting.builder()
-                        .service(new EchoService())
-                        .unary(EchoServiceGrpcProto.getDescriptor(),
-                                "EchoService",
-                                "Echo",
-                                Server::grpcEcho))
-                .build()
-                .start();
-    }
+    public static void main(final String[] args) {
+
+        try {
+
+            // Set the global configuration
+            final Config config = Config.create();
+            Config.global(config);
 
-    static void grpcEcho(EchoServiceGrpcProto.EchoRequest request, StreamObserver<EchoServiceGrpcProto.EchoResponse> responseObserver) {}
+            // Initialize the block storage, cache, and service
+            final BlockStorage<BlockStreamServiceGrpcProto.Block> blockStorage = new FileSystemBlockStorage(BLOCKNODE_STORAGE_ROOT_PATH_KEY, config);
+            final BlockStreamService blockStreamService = new BlockStreamService(1500,
```

Comment:
Should we Make a comment to set this `1500` as a configurable parameter in the future? so we don't forget.

Reply:
I added a TODO for it